galaxy.yml

```diff
@@ -7,7 +7,7 @@ authors:
 description: null
 license: GPL-3.0-or-later
 license_file: COPYING
-tags: null
+tags: [amazon, aws, cloud, networking]
```

Comment:
I'm not sure networking makes sense here; can you help me understand this tag's inclusion?

Reply:
Thanks @jillr for reviewing.
I was looking at another example when I typed that.
I'll submit an update to remove it.